src/v/cluster/self_test/cloudcheck.cc

```diff
@@ -41,7 +41,9 @@ cloudcheck::run(cloudcheck_opts opts) {
 
```

Comment:
> cluster/self_test: fix nits
Logging levels, duration values, and checking remote api
is initialized before use.

I think these changes can be folded back into the appropriate previous commit?